eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -344,6 +344,12 @@ jobs:
           extraBuildProperties="$extraBuildProperties ${{ parameters.extraProperties }}"
         fi
 
+        # Needed to diagnose https://github.com/dotnet/dnceng/issues/3305. These should be removed once the issue is resolved.
+        extraBuildProperties="$extraBuildProperties /p:Features=debug-analyzers"
+        export DOTNET_DbgEnableMiniDump=1
+        export DOTNET_DbgMiniDumpType=2
+        export DOTNET_DbgMiniDumpName=/vmr/artifacts/log/Release/%p.dmp
```

Comment:
I think you can eliminate this block and move the envvars https://github.com/dotnet/sdk/pull/42320/files#diff-dc549eeb99aa73960e74c9d4c160a3c76e3e414398bb348c1d5365aac0369ad5R349-R351 down to https://github.com/dotnet/sdk/pull/42320/files#diff-eeec14bcfae04e9276d77a77ecc85725588e875782ac908963dde2e3cf4d340aR140